javascript-modal/styles.css

```diff
@@ -0,0 +1,61 @@
+.container {
+  width: 1300px;
+  height: 900px;
+}
+
+.open-modal {
+  color: white;
+  background-color: royalblue;
+  font-size: 2rem;
+  padding: 5px 20px;
+  cursor: pointer;
+}
+
+.light {
+  background-color: white;
+}
+
+.dark {
+  background-color: darkgrey;
+}
+
+.no {
+  font-size: 2rem;
+  color: white;
+  background-color: limegreen;
+  padding: 10px 60px;
+}
+
+.modal {
+  display: none;
+  top: 200px;
+  left: 500px;
+  right: 500px;
+  height: 400px;
+  width: 700px;
+  position: absolute;
```

Comment:
You can simplify this a lot instead of using set px values. By setting all the offset values to 0, you will force the element to be centered.
```
position: fixed;
top: 0;
right: 0;
bottom: 0;
left: 0;
```